buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/BuildPlugin.groovy

```diff
@@ -314,6 +364,10 @@ class BuildPlugin implements Plugin<Project>  {
         compileTestJava.setSourceCompatibility('1.8')
         compileTestJava.setTargetCompatibility('1.8')
 
+        JavaCompile compileItestJava = project.tasks.getByName('compileItestJava') as JavaCompile
```

Comment:
We can set source and target compatibility on the project itself instead of doing it on each compile task individually.